kratos.gid/apps/DEM/write/writeMDPA_Walls.tcl

```diff
@@ -678,3 +455,40 @@ proc DEM::write::writeConditionMeshes2D { } {
     }
 }
 
+proc DEM::write::DefineMaterialTestConditions {group_node} {
+    if {[apps::getActiveAppId] eq "CDEM"} {
```

Comment:
This is not recommended by @jginternational . The GiD guys told me that we can use inheritance like in Python. So let's try it, this function can be overriden in CDEM.

Reply:
fixed. and tested.